### PR TITLE
Fix mispelling in hydra spec : writable to writeable

### LINKED ISSUE #3464: hydra:writable should be hydra:writeable (misspelled)
**API Platform version(s) affected**: any

**Description**  
There has never been `hydra:writable` in the Hydra spec.

**How to reproduce**  
N/A

**Possible Solution**  
Rename `hydra:writable` to `hydra:writeable`. We have to make the corresponding changes in https://github.com/api-platform/api-doc-parser too.

**Additional Context**  
N/A

### PR DESCRIPTION
**Pull request by adrientiburce**
|          Q            |            A          |
| ------------- | ------------- |
| Bug fix?          |       Yes            |
| New feature? |        no             |    
| BC breaks?     |        no             |   
| Deprecations? |        no            |   
| Tickets? |       fixes #3464              |   
     
PR on **api-doc-parser** : https://github.com/api-platform/api-doc-parser/pull/78

### CODE REVIEW

**Comment by adrientiburce** [comment]
Done :D

**Comment by alanpoulain** [comment]
It seems there is also `writableLink`.

File: src/Metadata/Extractor/YamlExtractor.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ private function extractProperties(array $resourceYaml, string $resourceName, st
             $this->resources[$resourceName]['properties'][$propertyName] = [
                 'description' => $this->phpize($propertyValues, 'description', 'string'),
                 'readable' => $this->phpize($propertyValues, 'readable', 'bool'),
-                'writable' => $this->phpize($propertyValues, 'writable', 'bool'),
+                'writeable' => $this->phpize($propertyValues, 'writeable', 'bool'),
```
Comment (alanpoulain) [inline src/Metadata/Extractor/YamlExtractor.php:107]: This is not linked to Hydra.

**Comment by adrientiburce** [comment]
I updated my PR, I think it's good now, 
do I also have to change the text in the behat feature : "property is writable"

File: src/Metadata/Extractor/YamlExtractor.php (hunk 1 above)
Comment (adrientiburce) [reply src/Metadata/Extractor/YamlExtractor.php]: I removed it

File: features/bootstrap/HydraContext.php [hunk 2]
```diff
@@ -192,8 +192,8 @@ public function assertPropertyIsNotReadable(string $propertyName, string $classN
      */
     public function assertPropertyIsWritable(string $propertyName, string $className)
     {
-        if (!$this->getPropertyInfo($propertyName, $className)->{'hydra:writable'}) {
-            throw new ExpectationFailedException(sprintf('Property "%s" of class "%s" is not writable', $propertyName, $className));
+        if (!$this->getPropertyInfo($propertyName, $className)->{'hydra:writeable'}) {
+            throw new ExpectationFailedException(sprintf('Property "%s" of class "%s" is not writeable', $propertyName, $className));
```
Comment (teohhanhui) [inline features/bootstrap/HydraContext.php:196]: No need to change this. Both `writable` and `writeable` are correct: https://en.wiktionary.org/wiki/writable

Comment (teohhanhui) [reply features/bootstrap/HydraContext.php:196]: >  do I also have to change the text in the behat feature : "property is writable"

No.

Comment (adrientiburce) [reply features/bootstrap/HydraContext.php]: Okay I removed it

**Comment by teohhanhui** [comment]
Thanks @adrientiburce! :tada:

**Pull request merged**